### ISSUE TITLE
Updating package-lock.json file after npm-update -u

### ISSUE THREAD

**Issue by davidtzoor** [dylang/npm-check#261]
Hi,

I noticed that the `package-lock.json` file isn't affected by the `npm-check -u` command, after installation is completed.

Is there a way to get `npm-check` update this file?

Thanks :)

**Comment by Ionaru** [comment]
`npm-check -u` does update `package-lock.json` for me, check if you're using `npm 5` or higher.

**Comment by LinusU** [comment]
Yeah, npm-check just executes `npm install ...` so the lockfile should be updated if you are using npm 5 or higher

**Comment by jsveron23** [comment]
It's not updating for me.

I am using `npm v6.8.0`

**Comment by mo** [comment]
Same thing happened for me. I upgraded wdio/cli from 8.39.1 to 9.1.0 but afterwards the transitive dependency import-meta-resolve was still pinned to 4.0.0 in package-lock.json even though 4.1.0 has been releated and the package.json for wdio/cli points to ^4.0.0.0  I also tried to delete node_modules and re-run npm install, and the lock file still pointed to import-meta-resolve 4.0.0. After that I deleted the lock file and re-ran "npm install" and even then I still had import-meta-resolve 4.0.0 in the newly generated package-lock.json. Finally, after deleting both node_modules dir and package-lock.json simultaneously and re-running "npm install" I finally got import-meta-resolve 4.1.0 pinned in the lock file. I'm using node 22 and npm 10.8.3.
